build.yml

```diff
@@ -0,0 +1,27 @@
+sdk: '2'
+language: bash
+go-runtime:
+  version: '1.10-alpine'
+native:
+  image: 'openjdk:8-jre-alpine'
+  static:
+      - path: src/main/sh/native.sh
+        dest: native
+  deps:
+      - 'apk add gradle'
```

Comment:
Is it a dependency for running the driver? Why?

Reply:
Vestigial, fixed on next commit.